.github/workflows/ci.yml

```diff
@@ -65,3 +65,82 @@ jobs:
       - name: Upload to codecov
         run: |
           bash <(curl -s https://codecov.io/bash)
+
+  sentry:
+    runs-on: ubuntu-latest
+    timeout-minutes: 25
+    steps:
+      # Checkout codebase
+      - name: Checkout snuba
+        uses: actions/checkout@v2
+
+      # Checkout Sentry and run integration tests against latest snuba
+      - name: Checkout sentry
+        uses: actions/checkout@v2
+        with:
+          repository: getsentry/sentry
+          path: sentry
+
+      - name: Build snuba docker image
+        run: |
+          docker pull getsentry/snuba:latest || true
+          docker build . \
+            --build-arg PYTHON_VERSION=3.8 \
+            -t getsentry/snuba \
+            --cache-from getsentry/snuba:latest \
+            --target testing
```

Comment:
I'm looking at the output of the step and the docs but I'm not understanding the use of `--target`.

Docs:
https://docs.docker.com/engine/reference/commandline/build/#specifying-target-build-stage---target

Reply:
We have some requirements that are used only for testing, so we use build targets in the dockerfile to specify when to install those or not.

Reply:
@evanh cool! Is it almost as a condition that needs to met in order to execute?